lib/snail.rb

```diff
@@ -35,6 +35,7 @@ class UnknownAttribute < ArgumentError; end
     :locality   => :city,
     :state      => :region,
     :province   => :region,
+    :county     => :region,
```

Comment:
> US counties aren't really used in mailing addresses

hmm, that's a good point. it may work out in practice? but let's try deferring on this alias -- it could be difficult to undo in future versions.